src/pages/Proposal/ProposalSummary/index.tsx

```diff
@@ -91,13 +100,16 @@ export const ProposalSummaryUi: React.FC<
 
   let updateProposalButton;
   let transactionHash = startTxHash;
+
   switch (state) {
     case 'Active':
       updateProposalButton = (
         <SecondaryButton
           onClick={handleCancelProposal}
           css={styles.updateProposalButton}
           loading={isCancelProposalLoading}
+          data-testid={TEST_IDS.proposalSummary.cancelButton}
```

Comment:
In this instance we can use the content directly to target the button since it doesn't include any string interpolation

Reply:
This is a valid solution. It is better than using the translation string because the text appears multiple times on the page. Using a testId allows the selection of the specific component we want to use